### ISSUE TITLE
Train CenterNet on both training set and validation set

### ISSUE THREAD

**Issue by Alien1007** [xingyizhou/CenterNet#520]
Thanks for your work！
If I want to train this model on both training set and validation set, will --trainval helps? Or I should just put two sets together manually and stop valuating during training?

**Comment by xingyizhou** [comment]
No it will not. You will need to create a merged json file. You can refer to [this](https://github.com/xingyizhou/CenterNet/blob/master/src/tools/merge_pascal_json.py).

**Comment by Alien1007** [comment]
Thanks a lot！
